src/redis-cli.c

```diff
@@ -5115,10 +5115,19 @@ static int clusterManagerFixMultipleSlotOwners(int slot, list *owners) {
         clusterManagerDelSlot(n, slot, 1);
         if (!clusterManagerSetSlot(n, owner, slot, "node", NULL)) return 0;
         if (count > 0) {
+            //the source node should set importing flag before moveslot
```

Comment:
The same logic exists in clusterManagerFixSlotsCoverage:'case "3": keys in multiple nodes'
It's might be worth unifying it into clusterManagerMoveSlot and specifying the behavior using an alternative to CLUSTER_MANAGER_OPT_COLD

Reply:
> The same logic exists in clusterManagerFixSlotsCoverage:'case "3": keys in multiple nodes'
> It's might be worth unifying it into clusterManagerMoveSlot and specifying the behavior using an alternative to CLUSTER_MANAGER_OPT_COLD

I think it is inappropriate to unify it into clusterManagerMoveSlot:
1. Function clusterManagerMoveSlot has already used the flag CLUSTER_MANAGER_OPT_COLD. If other flags added, it will affect this flag and make the logical judgment of the function more complicated.
2. This is not a routine operation and contains many subtle judgments. For example,  command  CLUSTER SETSLOT {SLOT} {NODE} {OWNER_NODE} is executed first, and it may be necessary to determine whether the number of keys contained in the node is greater than 0. These behaviors are not very consistent with clusterManagerMoveSlot.